website/src/pages/index.tsx

```diff
@@ -44,6 +44,33 @@ const ThemeResetComponent = () => {
   return (null);
 };
 
+// structured data for improved SEO
+const structuredData = {
+  "@context": "https://schema.org",
+  "@type": "softwareApplication",
+  name: "Apache APISIX",
+  alternateName: "APISIX",
+  description: "Open source and cloud native API gateway, based on the Nginx library and etcd.",
+  abstract: "High-performance, cloud-native, open source API Gateway from the Apache Software Foundation.",
+  url: "https://apisix.apache.org",
+  thumbnailUrl:
+    "https://raw.githubusercontent.com/apache/apisix/master/logos/apisix-white-bg.jpg",
+  sameAs: [
+    "https://twitter.com/ApacheAPISIX",
+    "https://github.com/apache/apisix",
+    "https://www.youtube.com/@apacheapisix",
+    "https://www.linkedin.com/company/apache-apisix/",
+  ],
+  isPartOf: "Apache Software Foundation",
+  maintainer: "Apache Software Foundation",
+  contributor: [
```

Comment:
What are the criteria to pick the contributors?

Reply:
I just went to orbit and found top organizations that contribute to APISIX. Should we do it?

Reply:
You need to have approval of some representative of the organizations for that.

Reply:
Removed the section.